terraform/gcp/locals.tf

```diff
@@ -2,4 +2,5 @@ locals {
   buz_sha               = "sha256:1083c0333c284dfa16dd7cc621f90b8a1197fe4d9905237e41f2f1a495481d92"
   buz_config_path       = "HONEYPOT_CONFIG_PATH"
   buz_config_path_value = "/etc/buz"
+  domain                = "honey.buz.dev"
```

Comment:
If we move this into a tfvar without a default so it can be provided upon `terraform apply` or within `.tfvars`?